custom/logistics/tasks.py

```diff
@@ -2,92 +2,96 @@
 from decimal import Decimal
 from functools import partial
 import itertools
-from couchdbkit import ResourceNotFound
+import celery
+from celery.canvas import chain
 from django.db import transaction
 from casexml.apps.stock.const import TRANSACTION_TYPE_LA
 from casexml.apps.stock.models import StockReport, StockTransaction
 from corehq.apps.commtrack.models import SupplyPointCase, update_stock_state_for_transaction
 from corehq.apps.locations.models import SQLLocation, Location
 from corehq.apps.products.models import SQLProduct
-from couchforms.models import XFormInstance
 from custom.logistics.commtrack import save_stock_data_checkpoint, synchronization
 from custom.logistics.models import StockDataCheckpoint
-from celery.task.base import task
```

Comment:
It's deprecated: http://docs.celeryproject.org/en/2.2/reference/celery.task.base.html